@types/phaser3-rex-plugins/plugins/behaviors/textedit/TextEdit.d.ts

```diff
@@ -0,0 +1,17 @@
+declare module 'phaser3-rex-plugins/plugins/behaviors/textedit/TextEdit' {
+  export default class TextEdit {
+    constructor(gameObject: Phaser.GameObjects.GameObject);
+    gameObject: Phaser.GameObjects.GameObject
+    scene: Phaser.Scene
+    inputText: string
```

Comment:
`inputText`, `onClose`, `delayCall` can be undefined, it's need to change to `inputText?`, ...